### ISSUE TITLE
About box points to misleading contributors page on Github (Trac #912)

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#995]
The About box says 'A list of individual contributors can be found at: https://github.com/orgs/SasView/people. Although this page exists, the only contributors listed are PDB and AJJ.

Either the Github page needs updating or the text in the About box needs replacing (eg, to point at http://www.sasview.org/contact.html).

Assigning this to PDB for a decision.

Migrated from http://trac.sasview.org/ticket/912
```json
{
    "status": "closed",
    "changetime": "2017-04-09T12:47:35",
    "_ts": "2017-04-09 12:47:35.321088+00:00",
    "description": "The About box says 'A list of individual contributors can be found at: https://github.com/orgs/SasView/people. Although this page exists, the only contributors listed are PDB and AJJ.\n\nEither the Github page needs updating or the text in the About box needs replacing (eg, to point at http://www.sasview.org/contact.html).\n\nAssigning this to PDB for a decision.",
    "reporter": "smk78",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Admin",
    "time": "2017-04-04T09:47:29",
    "component": "SasView",
    "summary": "About box points to misleading contributors page on Github",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.2.0",
    "owner": "ajj",
    "type": "defect"
}
```

**Comment by smk78** [comment]
Trac update at `2017/04/04 10:55:55`: 
* **smk78** commented:

Update

AJJ discovered that ALL contributors are only shown IF you are logged into Github.

Reassigning this ticket to AJJ to do some Github furtling.


* **smk78** changed owner from "butler" to "ajj"
* **smk78** changed status from "new" to "assigned"

**Comment by ajj** [comment]
Trac update at `2017/04/09 12:47:35`: 
* **ajj** commented:

In changeset ea45bfe4e68bda8abf0866aa2c51cfcf4cf63e36:

```text
#!CommitTicketReference repository="sasview" revision="ea45bfe4e68bda8abf0866aa2c51cfcf4cf63e36"
Changing link for list of contributors to point to website.

Website list needs to be updated, but this commit closes #995
```


* **ajj** changed resolution from "" to "fixed"
* **ajj** changed status from "assigned" to "closed"
